community/kernel/src/main/java/org/neo4j/kernel/impl/api/KernelStatement.java

```diff
@@ -187,14 +207,35 @@ final void forceClose()
         }
     }
 
-    private void cleanupResources()
+    final String authSubjectName()
     {
-        storeStatement.release();
-        // closing is done by KTI
+        return transaction.mode().name();
```

Comment:
we do not have an enum for that, no. in enterprise, we can have custom roles that would show up here